.vscode/launch.json

```diff
@@ -90,7 +90,7 @@
 				"**/winjs*.js"
 			],
 			"webRoot": "${workspaceFolder}",
-			"timeout": 15000
+			"timeout": 75000
```

Comment:
Did you mean to commit this change? 75 seconds seems like it might be too long in cases where Azure Data Studio is actually failing to start